sos/report/plugins/aap_gateway.py

```diff
@@ -35,10 +35,10 @@ def setup(self):
             "/etc/ansible-automation-platform/gateway/*.cert",
         ])
 
-        self.add_cmd_output([
-            "aap-gateway-manage list_services",
-            "ls -ll /etc/ansible-automation-platform/",
-            "ls -ll /etc/ansible-automation-platform/gateway/",
+        self.add_cmd_output("aap-gateway-manage list_services")
+        self.add_dir_listing([
+            "/etc/ansible-automation-platform/",
+            "/etc/ansible-automation-platform/gateway/",
```

Comment:
We could remove this now, if we're doing directory listing recursively by default

Reply:
Good eye!